hls4ml/backends/vivado/passes/convolution_templates.py

```diff
@@ -93,6 +93,11 @@ def format(self, node):
         else:
             params['fill_fn'] = 'FillConv1DBuffer'
 
+        if node.get_attr('filt_width') == 1 and node.model.config.get_config_value('IOType') == 'io_parallel':
```

Comment:
Is this needed? I assumed it was handled by the optimizer.

Reply:
In principle, maybe we don't have to check this explicitly again because this function will never be used unless it's a pointwise conv (and optimized as such). Do you suggest to remove it? I tried to follow the example about for other generated code like the fill_buffer, that does check for `io_parallel` at least.

Reply:
My suggestion was to remove it so if the pointwise optimizer was turned off, it would still work using the regular conv. Also, is the io_stream right?

Reply:
I don't think this particular piece of code prevents us from using standard conv if the optimizer can be skipped. This just adds a parameter to the config that may or may not be used.

However, I did try skipping the optimizer with this snippet: 
https://gist.github.com/jmduarte/351871db7fdfbf530ec02aecaa95e756#file-test_deepset-py-L48
```python
config['SkipOptimizers'] = ['vivado:optimize_pointwise_conv', 'vitis:optimize_pointwise_conv']
```
and I find that the optimizer still runs even though it seems to be successfully removed from the optimizer list (!). If I manually deactivate it (by having the matching criteria always return False), then it is skipped. Do we know why this optimizer is always running?

Reply:
SkipOptimizer feature doesn't work since the introduction of flows. We'll need a different way of exposing the feature to the user eventually. Right now the only way is to set the flows through the config and make sure the optimizer isn't there. It's a bit hacky.